docs/services/flows.rst

```diff
@@ -3,6 +3,26 @@ Globus Flows
 
 .. currentmodule:: globus_sdk
 
+The Flows service allows users to create automation workflows or, simply, "flows".
+When a flow is started, it must be authorized to perform actions on the user's behalf.
+
+Because a running flow can perform actions on the user's behalf,
+the Globus SDK has two client classes that can interact with the Flows service:
+a :class:`FlowsClient` and a :class:`SpecificFlowClient`.
+They differ in what scopes and user consents they require:
```

Comment:
Might help to be extra explicit that they do different things:

```suggestion
They differ in the operations they can perform as well as the scopes
and user consents they require:
```

Reply:
I think I've addressed this.